pkg/detector/detector.go

```diff
@@ -108,42 +109,61 @@ func (d *Detector) Run() error {
 				// Get the node details
 				result := checkableNodes[nodeName]
 				// do the check for this node
+				klog.V(4).Infof("about to check node %s with ip %s", nodeName, result.NetNode.IP)
 				nodeDown, err := isNodeDown(result.NetNode.IP)
 				// record the results
 				result.Err = err
 				result.IsNodeDown = nodeDown
+				if result.Err != nil {
+					klog.V(4).Infof("error checking node %s, %s", result.NetNode.IP, result.Err)
+				} else {
+					if nodeDown {
+						klog.V(4).Infof("node %s is unreachable, repeat NOT reachable", result.NetNode.IP)
```

Comment:
Nitpick, this log line and the one above should probably be warning level or higher?

Reply:
Line 121 is normal operation, just debugging as the node down is reported at level v1 later. - the one above I've made  an error though 👍